src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -14,54 +14,63 @@
 import codeu.chat.client.Controller;
+import codeu.chat.common.ConversationSummary;
+import codeu.chat.common.User;
+import codeu.chat.common.Message;
 import codeu.chat.client.View;
 import codeu.chat.util.Logger;
 
 public final class ChatGuiFX extends Application {
 
     private final static Logger.Log LOG = Logger.newLog(ChatGuiFX.class);
 
-
     private static final double WINDOW_WIDTH = 1000;
     private static final double WINDOW_HEIGHT = 500;
     private static final String SIGNIN_ERROR_MESSAGE = "Your username or password does not match. Have you signed up yet?";
     private static final String SIGNUP_ERROR_MESSAGE = "Sorry, that username already exists. Please choose a different one.";
     private static final String BADCHAR_ERROR_MESSAGE = "Usernames and passwords can only be composed of letters and numbers, no special characters.";
 
-
     // both page vars
     //private ClientContext clientContext;
 
+    /*
+        **This is something to think about**
+        // TODO: split the login & main screens into separate methods
+        public void run(String[] args) {
+            buildLoginScene();
+            buildMainScene();
+        }
+    */
 
     // login page vars
-    private Stage thestage;                         // Holds the scene that user is currently viewing
-    private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
+    // Holds the scene that user is currently viewing
+    private Stage thestage;
+    // Scenes to hold all the elements for each page
+    private Scene signInScene, mainScene;
+    // Takes input for username and password
     private TextField userInput;
-    private PasswordField passInput;                // Takes input for username and password
+    private PasswordField passInput;
 
     // There was merge conflict here. I have no idea why
-    private Label errorLabel;                       // Displays error messages
+    // Displays error messages
+    private Label errorLabel;
     private static ClientContext clientContext;
 
+    // holds the list of conversations
+    private ObservableList<String> convoList;
     // list of conversations
-    private ObservableList<String> convoList;      // list of conversations
-    private ListView<String> conversations;        // holds the list of conversations
-    Text chatTitle;                                // title of conversation
-
-    /*public void run(String[] args) {
-        try {
-            // launches gui
-            Application.launch(ChatGuiFX.class, args);
-        } catch (Exception ex) {
-            System.out.println("ERROR: Exception in ChatGuiFX.run. Check log for details.");
-            LOG.error(ex, "Exception in ChatGuiFX.run");
-            System.exit(1);
-        }
-
-    }*/
+    private ListView<String> conversations;
+    // title of conversation
+    private Text chatTitle;
+    // holds the list of messages
+    private ObservableList<String> messageList;
```

Comment:
Comments like this could be more descriptive. The variable name already tells me it's a list of messages. Where are these messages retrieved from? Are they messages for the whole system, or a single conversation, or a single user, or something else? Same thing applies to many of these variables.